presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -866,12 +866,12 @@ The following procedures are available:
 Extra Hidden Columns
 --------------------
 
-The Hive connector exposes extra hidden metadata columns in Hive tables. You can query these
-columns as a part of SQL query like any other columns of the table.
+The Hive connector exposes extra hidden metadata columns in Hive tables. Query these
+columns as a part of the query like any other columns of the table.
 
 * ``$path`` : Filepath for the given row data
-* ``$file_size`` : Filesize for the given row
-* ``$file_modified_time`` : Last file modified time for the given row
+* ``$file_size`` : Filesize for the given row (int64_t)
+* ``$file_modified_time`` : Last file modified time for the given row (int64_t)
```

Comment:
wouldn't hurt to define this one a little further. Is it seconds, milliseconds, or microseconds since the epoch? (or even something else?) But that can be for a later PR

Reply:
> wouldn't hurt to define this one a little further. Is it seconds, milliseconds, or microseconds since the epoch? (or even something else?) But that can be for a later PR

I could add that in this one. 

@aditi-pandit, @agrawalreetika: would either of you tell me what $file_modified_time is defined as so I can add it to the doc, please?

Reply:
@steveburnett : This should be seconds since epoch.

Reply:
Thanks @aditi-pandit and @agrawalreetika! I will add this in this PR.